flood_ml/data_utils_test.py

```diff
@@ -0,0 +1,37 @@
+import data_utils
+
+from numpy.testing import assert_array_equal
+import tensorflow as tf
+import unittest
+
+
+class TestDataUtils(unittest.TestCase):
```

Comment:
We've been writing pytest style tests where a test class and a unittest.main() call are unnescessary.

https://docs.pytest.org/en/8.0.x/#a-quick-example
https://github.com/UrbanSystemsLab/climateiq-cnn/blob/main/usl_pipeline/cloud_functions/main_test.py

Reply:
Fixed.